src/ir/iteration.h

```diff
@@ -84,10 +84,13 @@ class ChildIterator {
 
 // Returns true if the current expression contains a certain kind of expression,
 // within the given depth of BFS. If depth is -1, this searches all children.
-template<typename T> bool containsChild(Expression* parent, int depth = -1) {
+template<typename T> bool containsChild(Expression* curr, int depth = -1) {
```

Comment:
the name seems potentially misleading as it looks not just at children. how about just `contains`?

also, perhaps this could move to `find.h` alongside `FindAll`. It's similar to those except it returns a boolean and not a list.

Reply:
Moved it to find_all.h and changed the name to `contains` (which looks too common, but fortunately we don't have a name clash yet).